impls/jq/step5_tco.jq

```diff
@@ -503,21 +275,25 @@ def EVAL(env):
                     } | TCOWrap($res | last.env; $_orig_retenv; false)
                 end
             ) //
-            (select(.kind == "hashmap") |
+            (
+                select(.kind == "hashmap") |
                 [ { env: $_menv, list: .value | to_entries } | hmap_with_env ] as $res |
                 {
                     kind: "hashmap",
                     value: $res | map(.value) | from_entries
                 } | TCOWrap($res | last.env; $_orig_retenv; false)
             ) //
-            (select(.kind == "function") |
+            (
+                select(.kind == "function") |
                 . | TCOWrap($_menv; $_orig_retenv; false) # return this unchanged, since it can only be applied to
             ) //
-            (select(.kind == "symbol") |
+            (
+                select(.kind == "symbol") |
                 .value | env_get($_menv) | TCOWrap($_menv; null; false)
```

Comment:
The other half of REGRESS failures are not your fault, they were here from before.
This should be
```suggestion
                .value | env_get($_menv) | TCOWrap($_menv; $_orig_retenv; false)
```
(as the evaluation of a symbol does not affect the resulting env)